### PR TITLE
adds svgicon components and tabs

### PR DESCRIPTION
**Pull request by f1lander**
### issues related

#5 #7

### CODE REVIEW

File: components/atoms/icons/IconArrowForward.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { SvgIcon, SvgIconProps } from "../SvgIcon/SvgIcon";
+
+export const IconArrowForward = (props: SvgIconProps) => (
+  <SvgIcon {...props}>
+    <path
+      d="M8.99992 0.333496L8.05992 1.2735L11.1133 4.3335H0.333252V5.66683H11.1133L8.05325 8.72683L8.99992 9.66683L13.6666 5.00016L8.99992 0.333496Z"
```
Comment (xenowits) [inline components/atoms/icons/IconArrowForward.tsx]: discussion: what does the `d` attribute signify?

**Review by xenowits — APPROVED** [review]
Looks good! It'd be great to add comments wherever possible.

**Comment by xenowits** [comment]
Also, adding a licence would resolve yarn warnings

File: components/atoms/icons/IconArrowForward.tsx (hunk 1 above)
Comment (f1lander) [reply components/atoms/icons/IconArrowForward.tsx]: @xenowits here -> https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d 

This is the svg, generated from figma, I just copy as svg from figma.

**Pull request merged**